02_array_Liakh

zhlyakh opened the pull request:
ну как-то так...

zhlyakh committed:
02_array_Liakh

Constantinehub commented on script.js:
```diff
@@ -0,0 +1,7 @@
+let firstArray = [];
+// console.log(typeof firstArray);
+firstArray = [98, 'Terve kaikille', false, 123n, undefined, null, {age:18, height:163, sex:'female'}, Symbol('Bio')];
+
+firstArray.push(+prompt('Введите число'));
+firstArray.push(String(firstArray[8]));
+console.log(firstArray);
```
Здесь ты немного не правильно понял задачу. Суть в том чтобы не просто перечислить в массиве все типы данных, а сделать это в динамике, каждый раз спрашивая пользователя ввести соответствующие данные и последовательно добавляя каждый новый ответ пользователя в массив где в конце всех операций получившийся массив выводится в консоль со всем что получилось.

zhlyakh commented:
Хорошо, переделаю

вт, 4 мая 2021 г., 15:33 Constantine ***@***.***>:

> *@Constantinehub* commented on this pull request.
> ------------------------------
>
> In script.js
> <https://github.com/Constantinehub/FrontendPro_HW/pull/3#discussion_r625741118>
> :
>
> > +firstArray = [98, 'Terve kaikille', false, 123n, undefined, null, {age:18, height:163, sex:'female'}, Symbol('Bio')];
> +
> +firstArray.push(+prompt('Введите число'));
> +firstArray.push(String(firstArray[8]));
> +console.log(firstArray);
>
> Здесь ты немного не правильно понял задачу. Суть в том чтобы не просто
> перечислить в массиве все типы данных, а сделать это в динамике, каждый раз
> спрашивая пользователя ввести соответствующие данные и последовательно
> добавляя каждый новый ответ пользователя в массив где в конце всех операций
> получившийся массив выводится в консоль со всем что получилось.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Constantinehub/FrontendPro_HW/pull/3#pullrequestreview-651181925>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT25ABOT2ZFDE3CXJHOXQ2TTL7SSRANCNFSM44CIKM6A>
> .
>

zhlyakh committed:
02_array_Liakh

Вот тут я слегка видоизменил. Я изначально создал массив из 4 элементов - которые трудно задать или привести в них. Но потом, по ходу промптов, он постепенно наполняется текстовым, булевым, числовым значением и объектом.
